spec/csa/ccm/cli/command_spec.rb

```diff
@@ -6,9 +6,9 @@
   end
 
   after(:all) do
-    # FileUtils.rm_rf './tmp'
-    # FileUtils.rm_f './caiq-3.0.1.yaml'
-    # FileUtils.rm_f './resources/csa-caiq-v3.0.1-12-05-2016.yaml'
+    FileUtils.rm_rf './tmp'
```

Comment:
Can we use:

```ruby
require 'tmpdir'

Dir.mktmpdir {|dir|
  # use the directory...
  open("#{dir}/foo", "w") { ... }
}
```

This ensures we:
1. don't pollute the current directory
2. don't have to clean up

Reply:
This will partially work because according to requirements to command requirements, some output must be placed alongside with input files

Reply:
@CAMOBAP795 I don't quite understand the issue:

1. This comment was only about the specs -- if it had to create a `tmp/` directory, might as well use `Dir.mktmpdir`.

2. For `-o` option, if `-o` wasn't specified, create the output in the same path (same base name but different extension). If `-o` specified, the output should be `{current directory where command was run}/{-o path}`.

Reply:
@ronaldtse exactly but when you run `xlsx2yaml ./resources/csa-caiq-v3.0.1-12-05-2016.xlsx` it means that output YAML will be placed to `./resources/csa-caiq-v3.0.1-12-05-2016.yaml`

BTW I removed such duplicate test cases and now every generated file placed in `./tmp`

Please check current implementation

Reply:
I see. But when we test, we should probably copy `./resources/csa-caiq-v3.0.1-12-05-2016.xlsx` to a temp directory, and the run the command there?

Thanks!

Reply:
@ronaldtse right simple/good approach, thanks, I will do this way

Reply:
For `Dir.mktmpdir`, I really don't want to make a `./tmp`. It should be something like this:

```ruby
Dir.mktmpdir {|dir|
  # use the directory...
  open("#{dir}/foo", "w") { ... }
}
```

Or even
```
let(:tmpdir) { Dir.mktmpdir }
```

Reply:
Done